tests/yast2_gui/yast2_hostnames.pm

```diff
@@ -17,37 +17,50 @@
 use base "y2x11test";
 use strict;
 use testapi;
-use utils qw(type_string_slow clear_console);
+use utils 'clear_console';
+use x11utils qw(turn_off_kde_screensaver turn_off_gnome_screensaver);
 
 sub run {
-    my $self   = shift;
-    my $module = "host";
-
+    my $self         = shift;
+    my $module       = "host";
+    my $dm           = lc get_var('DESKTOP');
+    my $hosts_params = {
+        ip    => '195.135.221.134',
+        fqdn  => 'download.opensuse.org',
+        alias => 'download-srv'
+    };
+    my %execute = (
+        gnome => \&turn_off_gnome_screensaver,
+        kde   => \&turn_off_kde_screensaver
+    );
+    $execute{$dm}->();
     select_console 'root-console';
     #	add 1 entry to /etc/hosts and edit it later
     script_run "echo '80.92.65.53    n-tv.de ntv' >> /etc/hosts";
     clear_console;
     select_console 'x11';
     $self->launch_yast2_module_x11($module, match_timeout => 90);
     assert_and_click "yast2_hostnames_added";
-    wait_still_screen 1;
-    wait_screen_change { send_key 'alt-i'; };
-    send_key 'alt-t';
-    type_string 'download-srv';
-    wait_still_screen 1;
-    send_key 'alt-h';
-    type_string 'download.opensuse.org';
-    wait_still_screen 1;
     send_key 'alt-i';
-    type_string_slow '195.135.221.134';
-    assert_and_click 'yast2_hostnames_changed_ok';
+    assert_screen 'yast2_hostnames_edit_popup';
+    send_key 'alt-i';
+    type_string($hosts_params->{ip}, max_interval => 13, wait_still_screen => 0.05, timeout => 5, similarity_level => 38);
+    send_key 'tab';
+    type_string($hosts_params->{fqdn}, max_interval => 13, wait_still_screen => 0.05, timeout => 5, similarity_level => 38);
+    #unless (wait_still_screen(stilltime => 0.05, timeout => 3, similarity_level => 40)) {
```

Comment:
please not commit commented code

Reply:
Thanks, Anton. I will remove it once I know what happens with pre-requisite PR.